### DISCUSSION TITLE
How to put in log string + int?

### DISCUSSION THREAD

**Discussion by maksglukhov** [gabime/spdlog#2120] [Q&A] [answered]
Hi guys,

I need to put in logs message + int 
I tried this but there is no int in logs and my text is cuted :
 auto mylogger = spdlog::basic_logger_mt("logger", "logs/logs.txt");
    mylogger->info("The number to guess is: " + ran);

how should i do?
Thank you!

**Comment by tt4g** [comment, marked as answer]
fmt library is used to format log messages with arguments in spdlog.
Try it.

```c++
auto mylogger = spdlog::basic_logger_mt("logger", "logs/logs.txt");
mylogger->info("The number to guess is: {}", ran);
```

And see fmt documentation: https://fmt.dev/8.0.0/

**Reply by maksglukhov** [reply]
Thank you! It works and I wil read the documentation.
